chart/kubeapps/templates/apprepository-deployment.yaml

```diff
@@ -23,6 +23,7 @@ spec:
       containers:
       - name: controller
         image: {{ template "kubeapps.image" (list .Values.apprepository.image .Values.global) }}
+        imagePullPolicy: IfNotPresent
```

Comment:
is this needed? I mean, if it's not present it will pull it anyways?

Reply:
not sure why we need this, the default k8s behaviour is to set to `Always` if the `latest` tag is used, or `IfNotPresent` otherwise.

Reply:
again, this was for dev purposes, I changed this to use my locally build image.